MobileWallet/TariLib/Wrappers/Utils/Migrations.swift

```diff
@@ -96,14 +102,21 @@ class Migrations {
         TariLogger.verbose("Migrated Tor cache to shared app group storage")
     }
 
+    static func setBackupPasswordToKeychain(password: String) {
+        sharedKeychainGroup.set(password, forKey: passwordKey)
+    }
+
+    static func loadBackupPasswordFromKeychain() -> String? {
+        return sharedKeychainGroup.string(forKey: passwordKey)
+    }
+
+    static func removeBackupPasswordFromKeychain() {
+        sharedKeychainGroup.removeObject(forKey: passwordKey)
+    }
```

Comment:
I'm not sure if these are migration functions - the placing may be wrong. @Jasonvdb

Reply:
Yeah, @SerhiiShovkoplias not really related to migrations